src/appleseed/foundation/platform/python.h

```diff
@@ -39,15 +39,15 @@
 #endif
 
 // Boost headers.
-#include "boost/python/detail/wrap_python.hpp"  // has to be first, to avoid redefinition warnings
 #include "boost/python.hpp"
+#include "boost/python/detail/wrap_python.hpp"  // has to be first, to avoid redefinition warnings
```

Comment:
This change should be reverted I think.

My guess is that you wrote a Python script to reorder the `#include` statements? ;) If so, could you share it?

Reply:
I used the sort includes script in the repo. But it does not work very well in some cases.
For example it messes with _begin_xxx and _end_xxx headers.